tools/src/versioning/android/index.ts

```diff
@@ -608,6 +616,15 @@ async function exportReactNdksIfNeeded() {
   }
 }
 
+async function addBuildGradleRenameTaskForPrebuiltLibs(version: string) {
+  const abiVersion = version.replace(/\./g, '_');
+  const filePath = path.join(versionedReactAndroidPath, 'build.gradle');
+  let content = await fs.readFile(filePath, 'utf8');
+  const renameTask = `        rename '(.+).so', '$$1_abi${abiVersion}.so'\n`;
+  content = content.replace(/(into "\$thirdPartyNdkDir\/hermes"\n)(\s+?\})/gm, `$1${renameTask}$2`);
+  await fs.writeFile(filePath, content);
```

Comment:
Try using `transformFileAsync` function from `Transforms.ts` — it would make this code cleaner 😉